bin/cdklocal

```diff
@@ -3,6 +3,8 @@
 const provider = require('aws-cdk/lib/api/aws-auth/sdk-provider');
 const { CdkToolkit } = require('aws-cdk/lib/cdk-toolkit');
 const { SDK } = require('aws-cdk/lib/api/aws-auth/sdk');
+const bootstrap = require("aws-cdk/lib/api/bootstrap");
```

Comment:
Getting below error on executing `cdklocal bootstrap` command. Working on a workaround to fix this error.
`lib_1.Bootstrapper is not a constructor`

Reply:
Fixed.